v-next/hardhat/src/internal/builtin-plugins/solidity-test/types.ts

```diff
@@ -0,0 +1,17 @@
+import type { SuiteResult } from "@ignored/edr";
+import type { Readable } from "node:stream";
+
+export type TestStatus = "Success" | "Failure" | "Skipped";
```

Comment:
I think EDR changed how they export enums now, so this can probably be improved in a later PR, so that we avoid the duplication.